nifi-nar-bundles/nifi-accumulo-bundle/nifi-accumulo-processors/src/main/java/org/apache/nifi/accumulo/processors/ScanAccumulo.java

```diff
@@ -254,7 +258,12 @@ public void process(final InputStream in, final OutputStream out) throws IOExcep
 
                             try{
                                 final RecordSchema writeSchema = writerFactory.getSchema(flowAttributes, new KeySchema());
-                                try (final RecordSetWriter writer = writerFactory.createWriter(getLogger(), writeSchema, out)) {
+                                List<RecordField> fieldList = new ArrayList<>();
+                                fieldList.addAll(writeSchema.getFields());
+                                fieldList.add(new RecordField("value", RecordFieldType.STRING.getDataType()));
```

Comment:
The Accumulo `Value` object contains a byte array which can be converted to UTF-8 String, but does conversion to String cover all use cases?

Reply:
Thanks for your help exceptionfactory, String will cover most cases